### PR TITLE
Added a basic template for frequently asked questions to the support …

### PR DESCRIPTION
**Pull request by AmazingConnie**
…page

- Rohan Mallu

#### Overall Review of Changes:
I added an accordion component to the support page for frequently asked questions.

##### Reason for Change
The support page can serve as the website's FAQ/Troubleshooting area.

##### Additional Changes
Changed next.js version to latest in package.json.

#### Tested:
Yes
unipathsupport

### CODE REVIEW

**Comment by rajeshrah22** [comment]
Hey Rohan, can you add a screenshot. 

Thanks 👍

**Comment by Gurpranked** [comment]
The build is failing because there is a version mismatch for Next JS. Could you please update your Next JS version in your code?

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "@mui/material": "^5.14.18",
     "@mui/material-nextjs": "^5.15.7",
     "aws-amplify": "^6.0.13",
-    "next": "latest",
+    "next": "^14.1.1-canary.56",
```
Comment (Gurpranked) [inline package.json:21]: This seems to be the cause of the issue. Could you explain why this is set to a Canary Build?

Comment (AmazingConnie) [reply package.json:21]: When I tested it the first time, I was running into the "Element type is invalid" error. The error page also said my version of Next.js was out of date. So I upgraded it. I did not realize Canary was a test build. Is there a way to fix this?

Comment (Gurpranked) [reply package.json:21]: Setting next to "latest" within package.json should set it to the right version. Let me know if that works or if something else comes up.

**Comment by Gurpranked** [comment]
There seems to be some backend issues with version mismatches on some of the dependencies. Will look into the backend to make sure about that. Currently the generated preview seems to be missing the changes made in code, even though they are present in source. Again, likely a backend issue and will be looked into.

**Comment by MartinMarwad** [comment]
Going to hold off on merging for now. Need to create a new branch with these changes without changes to `package.json` and `package-lock.json` files, and then make new PR for that.

**Review by rajeshrah22 — APPROVED** [review]
Very nice.

**Comment by rajeshrah22** [comment]
This has been moved to another pull request, @AmazingConnie your changes have now been merged to main from another branch.

**Pull request closed without merging**